core/utils.go

```diff
@@ -77,32 +78,33 @@ func ValidateStruct(param interface{}, paramName string) error {
 	return nil
 }
 
-// StringPtr : return pointer to string literal
+// StringPtr return pointer to string literal.
```

Comment:
To maintain the style you are using for these comments, you need: "return" -> "returns a"

Also for several methods below.